rclcpp/include/rclcpp/publisher.hpp

```diff
@@ -206,11 +230,11 @@ class Publisher : public PublisherBase
       ipm.message_sequence = message_seq;
       auto status = rcl_publish(&intra_process_publisher_handle_, &ipm);
       if (RCL_RET_PUBLISHER_INVALID == status) {
-        rcl_reset_error();  // next call will reset error message if not context
+        rcl_reset_error();  /* next call will reset error message if not context */
         if (rcl_publisher_is_valid_except_context(&intra_process_publisher_handle_)) {
           rcl_context_t * context = rcl_publisher_get_context(&intra_process_publisher_handle_);
           if (nullptr != context && !rcl_context_is_valid(context)) {
-            // publisher is invalid due to context being shutdown
+            /* publisher is invalid due to context being shutdown */
```

Comment:
@ivanpauno why the change in comment syntax? I'd rather stick to the previous one.